src/admin/components/ResourceTabs/InformationTab/components/ReportInformationAside/QuickActions.tsx

```diff
@@ -28,6 +29,11 @@ const ReportQuickActions: FC<QuickActionsProps> = ({ type }) => {
     return `/${view}?${queryParams}`;
   };
 
+  const { data: reportTaskUuid } = useGetV2TasksIDId(
+    { pathParams: { id: record?.taskId! } },
+    { enabled: !!record?.taskId }
+  );
```

Comment:
I'd rather we include the report task UUID on the v3 response than require an extra API request here. Should be pretty simple to add the task as a join into the project report (although maybe we haven't defined the Tasks table yet in v3? If that's the case, you can make it a very simple stub that just exposes the UUID; I'll be getting to the whole task definition soon).